contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,303 @@ availability:
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you'll use this feature flag key to check which experiment variant the user has been assigned to. 
 
-#### 2. Participant selection and experiment goal/metric
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](// TODO add link here) 
 
-Next, we want to select our experiment's universe. Here, we've decided that for our sign up button experiment we want to test our hypothesis on users from Asia.
+### Experiment variants
 
-We also want to set our experiment's "goal" now for which metric we want to track. Since we're testing whether the number of sign up button clicks increase or not, we'll use the "Trend" goal to measure aggregate values
+By default, all experiments have a 'control' and 'test' variant. If you have more variants that you'd like to test, you can use this field to add more variants and run multivariant tests.
 
-![Participants and goal metric](../../images/features/experiments/create-experiment-participants-goals.png)
+### Participant type
 
-If we want to test a hypothesis for a conversion rate metric, we'll use the "Funnel" goal instead.
+If you've created [groups](/docs/product-analytics/group-analytics) in PostHog, you can use this field to run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level. 
 
-#### 3. Experiment preview and minimal acceptable improvement threshold
+### Participant targeting
 
-Finally, in our experiment creation form, look to the upper right corner for our experiment's preview!
-We can see recommendations for run times and other experiment factors.
+By default, your experiment will target 100% of participants. If you'd like to target a more specific set of participants, or change rollout percentage, you'll need to do this on the feature flag that will be used by your experiment.
 
-![Experiment preview](../../images/features/experiments/create-experiment-preview.png)
+Below is a video showing this:
 
-We can also increase or decrease our minimal acceptable improvement depending on how much we care for our test's significance relevant to our other experiment factors.
+// TODO Add video of how to do this
 
-Perhaps we don't want to wait 278 days for our experiment to run and we'd prefer less precise results in favor of a shorter run time. We can simply bump the threshold to a higher percentage.
+### Experiment goal
 
-![Experiment preview significance change](../../images/features/experiments/create-experiment-preview-significance.png)
+Setting your goal metric will enable PostHog to automatically keep track of your goal metrics and let you know if your results are statiscally signifcant. You can select between a "trend" or "conversion funnel" goal.
```

Comment:
```suggestion
Setting your goal metric enable PostHog to calculate the impact of your experiment and if your results are [statistically significant](/docs/experiments/significance). You can select between a "trend" or "conversion funnel" goal.
```
What is the difference between trend and conversion funnel? When would you want to use one vs the other?

Reply:
Aside: We always want to push users towards funnel experiments; they're easier to setup; results are more reliable; and harder to get wrong. Trends have too many gotchas we've seen people fall for.